components/card.css

```diff
@@ -0,0 +1,120 @@
+.card {
+    width: 17rem;
+    background-color: #fff;
+    padding: 1rem;
+    border-radius: 5px;
+    position: relative;
+}
+
+.card-header {
+    display: flex;
+    align-items: center;
+    border-bottom: 2px solid var(--light-grey-color);
+    padding-bottom: 0.5rem;
+}
+
+.close {
+    margin-left: auto;
+    font-size: 1.4rem;
+}
+
+.card-title {
+    font-weight: 700;
+}
+
+.card-body {
+    padding-top: 0.5rem;
+}
+
+.card-img-container {
+    position: relative;
+}
+
+.card-img {
+    width: 100%;
+    border-radius: 2px;
+}
+
+.card-wishlist-icon {
+    position: absolute;
+    font-size: 2rem;
+    top: 10px;
+    right: 10px;
+    background-color: white;
+    border-radius: 50%;
+    padding: 0.5rem;
+    cursor: pointer;
+}
+
+.card-wishlist-icon:hover {
+    color: #e8415f;
+}
+
+.item-price {
+    font-weight: 700;
+    font-size: 1.2rem;
+}
+
+.card-discription {
+    padding-top: 5px;
```

Comment:
description

Reply:
Thanks for reviewing. I had fixed this typo.